base/stats.go

```diff
@@ -58,6 +70,8 @@ type SgwStats struct {
 	dbStatsMapMutex sync.Mutex
 }
 
+var SyncGatewayStats = *NewSyncGatewayStats()
```

Comment:
@JRascagneres do we still need to do `expvar.Publish(StatsGroupKeySyncGateway, &SyncGatewayStats)` ? Or is that handled by the stats framework itself now?